kravemir/GradleSassPlugin
# Update Gradle wrapper to v3.0

Gradle 3.0 will be faster for users and this will help ensure
future compatibility of the plugin.

Testing done:
- Build and publish plugin locally
- Run all 3 examples and check output
